cms/templates/js/container-access.underscore

```diff
@@ -0,0 +1,31 @@
+<%
+var selectedGroupsLabel = userPartitionInfo['selected_groups_label'];
+var selectedPartitionIndex = userPartitionInfo['selected_partition_index'];
+var category = this.model.attributes.category;
+var blockType = "";
+if (category == "split_test") {
+    blockType = "component";
+} else {
+    blockType = "unit"
```

Comment:
Can we use "unit" for the unit page and "component" so everything else so this will work with more than just split_test?